requirements_test.txt

```diff
@@ -52,3 +53,4 @@ types-PyYAML==6.0.12.20240311
 types-requests==2.31.0.3
 types-xmltodict==0.13.0.3
 uv==0.2.13
+zeroconf==0.132.2
```

Comment:
This probably means something else is wrong; adding this here, isn't right.

`zeroconf` is only a dependency of the `zeroconf` integration, not core.